CONTRIBUTING.md

```diff
@@ -66,75 +66,60 @@ Then every time you want to test a new version, you can just:
 ```shell
 make oci
 # Delete the mcoa pod which will make the Deployment pull the new image
-oc -n open-cluster-management delete pod -l app=multicluster-observability-addon-manager
+oc -n open-cluster-management-observability delete pod -l app=multicluster-observability-addon-manager
 ```
 
-### Disabeling specific signals 
+### Enable specific Observability Capabilities
 
-The addon supports disabling signals using the resource `AddOnDeploymentConfig`. For instance, to disable the logging signal create the following resource on the hub cluster:
+The addon supports enabling observability capabilities using the resource `AddOnDeploymentConfig`. For instance, to enable platform and user workloads logging/tracing/instrumentation create the following resource on the hub cluster:
 
 ```yaml
 apiVersion: addon.open-cluster-management.io/v1alpha1
 kind: AddOnDeploymentConfig
 metadata:
   name: multicluster-observability-addon
-  namespace: open-cluster-management
+  namespace: open-cluster-management-observability
 spec:
   customizedVariables:
-    - name: loggingDisabled
-      value: "true"
+  # Platform Observability
+  - name: platformLogsCollection
+    value: clusterlogforwarders.v1.logging.openshift.io
+  # User Workloads Observability
+  - name: userWorkloadLogsCollection
+    value: clusterlogforwarders.v1.logging.openshift.io
```

Comment:
What are the semantics of the value? 

In the proposal https://hackmd.io/aBUzPTEZRuCPp_kZqm4x2A?view it was agreed to support multiple stacks for a single capability.

Reply:
I would prefer to document it here, that the value can be a list

Reply:
@pavolloffay My intention was to add on the code base what we will start supporting now and move the hack.md and all the agreements made there into a formal ADR document. Is that ok for you?

Reply:
> What are the semantics of the value?

I picket the FQDN version over the simplified version to keep the path for version migrations open. Some of our supported APIs are declared as alpha, other as beta while other as stable. My intention is here for leave room for individual contributors to move at their own pace. AFAIU opentelemetry APIs are moving currently at a faster and innovative pace than others.